configs/fp16/metafile.yml

```diff
@@ -0,0 +1,85 @@
+Collections:
+  - Name: FP16
+    Metadata:
+      Training Data:
+        - KITTI
+        - nuScenes
+      Training Techniques:
+        - AdamW
+      Training Resources: 8x TITAN Xp
+      Architecture:
+        - Hard Voxelization
+        - Focal Loss
+    Paper:
+      - https://www.mdpi.com/1424-8220/18/10/3337
+      - https://arxiv.org/abs/1812.05784
+    README: configs/fp16/README.md
+
+Models:
+  - Name: hv_second_secfpn_fp16_6x8_80e_kitti-3d-car
+    In Collection: FP16
+    Config: configs/fp16/hv_second_secfpn_fp16_6x8_80e_kitti-3d-car.py
+    Metadata:
+      train time (s/iter): 0.84
+      Training Memory (GB):
+        - FP32: 5.4
+        - FP16: 2.9
+    Results:
+      - Task: 3D Object Detection
+        Dataset: KITTI
+        Metrics:
+          FP32 mAP: 79.07
+          FP16 mAP: 78.72
+    Weights: https://download.openmmlab.com/mmdetection3d/v0.1.0_models/fp16/hv_second_secfpn_fp16_6x8_80e_kitti-3d-car/hv_second_secfpn_fp16_6x8_80e_kitti-3d-car_20200924_211301-1f5ad833.pth
+
+  - Name: hv_second_secfpn_fp16_6x8_80e_kitti-3d-3class
+    In Collection: FP16
+    Config: configs/fp16/hv_second_secfpn_fp16_6x8_80e_kitti-3d-3class.py
+    Metadata:
+      train time (s/iter): 0.98
+      Training Memory (GB):
+        - FP32: 5.4
+        - FP16: 2.9
+    Results:
+      - Task: 3D Object Detection
+        Dataset: KITTI
+        Metrics:
+          FP32 mAP: 64.41
+          FP16 mAP: 67.4
+    Weights: https://download.openmmlab.com/mmdetection3d/v0.1.0_models/fp16/hv_second_secfpn_fp16_6x8_80e_kitti-3d-3class/hv_second_secfpn_fp16_6x8_80e_kitti-3d-3class_20200925_110059-05f67bdf.pth
+
+  - Name: hv_pointpillars_secfpn_sbn-all_fp16_2x8_2x_nus-3d
+    In Collection: FP16
+    Config: configs/fp16/hv_pointpillars_secfpn_sbn-all_fp16_2x8_2x_nus-3d.py
+    Metadata:
+      train time (s/iter): 2.14
+      Training Memory (GB):
+        - FP32: 16.4
+        - FP16: 8.37
+    Results:
+      - Task: 3D Object Detection
+        Dataset: nuScenes
+        Metrics:
+          FP32 mAP: 35.17
```

Comment:
FP32 AP is just a reference of this model and is produced by its FP32 version. So we do not need to add it